plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtMigrateCommandWrapper.java

```diff
@@ -144,6 +148,14 @@ Use VIR_DOMAIN_XML_SECURE (value = 1) prior to v1.0.0.
             // delete the metadata of vm snapshots before migration
             vmsnapshots = libvirtComputingResource.cleanVMSnapshotMetadata(dm);
 
+            // Verify Format of backing file
+            for (DiskDef disk : disks) {
+                if (disk.getDeviceType() == DiskDef.DeviceType.DISK
```

Comment:
Could this break on older CentOS/Ubuntu KVM hosts where the backfile is not supported?